firefox-ios/firefox-ios-tests/Tests/XCUITests/ActivityStreamTest.swift

```diff
@@ -3,6 +3,7 @@
 // file, You can obtain one at http://mozilla.org/MPL/2.0/
 
 import XCTest
+import Shared
```

Comment:
Why do we need to import `Shared`?

Reply:
Because the launch arguments comes from the Shared package. I believe before we didn't have to do it since there was a bridging header who was importing shared everywhere in the tests.